self_teach_challenge_1.1/script.js

```diff
@@ -21,18 +71,23 @@ const COUNTDOWN = () => {
     interval = setInterval(() => {
         if (countdownSeconds > 0) {
             countdownSeconds = countdownSeconds - 1;
+            minutesInput.setAttribute('value', (Math.floor(Number(countdownSeconds/60))))
+            secondsInput.setAttribute('value',countdownSeconds%60)
+            minutesInput.value = Math.floor(Number(countdownSeconds/60));
+            secondsInput.value = Number(countdownSeconds%60);
+            oneNumber();
```

Comment:
Do: You should either set the input values here, or in the `oneNumber` function, but not both, otherwise it's more error-prone and can introduce bugs. It would be cleaner to pass these to `oneNumber` as parameters.